### DISCUSSION TITLE
Newest Centaur not works

### DISCUSSION THREAD

**Discussion by Josefino** [EdNekebno/DGTCentaurMods#161] [General]
I have a two month old DGT Centaur, I have tried a different OS, but unfortunately, the program does not boot, and the display does not show anything. Everything seems fine, and the install script runs without errors. The LEDs on the chessboard are still blinking. Dirk's modification works even when it doesn't have the options that this one does.

**Comment by wormstein** [comment]
Hi, first of all, dirk is not a member of this project so we don't know how to relate to his changes. He is distributing a fork of this project (well, part of it...)

I think your probem is actually a common bug that we are seeing lately in which the 64 bit version of the Raspbian OS is not supported anymore. Please use the card-setup-tool again and install the 32 bit version of Raspberry Pi OS Lite.

I hope this bug will be fixed somehow soon.

Cheers,

**Comment by wormstein** [comment]
If dirk's fork is working on your new board, most likely the mod will work too because dirk's fork uses the same protocol to communicate with the board as our mod.

It just looks like the mod didn;t complete the self-setup sequence once the card was plugged in the board. This step usually takes 25-30 mins if the card is setrup correctly.

**Comment by Josefino** [comment]
Thank you for your prompt reply. I did test OS lite 32 bit, but I certainly didn't wait 25-30 minutes, rather I shut it down after 10-15 minutes, that may be the problem. I will try again then.

**Comment by Josefino** [comment]
Great! Finally, the software works and the menu appears. There was probably also a problem that chessboard was not in WiFi range. Thank you for your help. Josef
